ghost/members-api/lib/controllers/RouterController.js

```
@@ -478,6 +478,13 @@ module.exports = class RouterController {
                 ...data
             });
         } else if (type === 'donation') {
+            const rawText = req.body.personalNoteText;
+            if (typeof rawText !== 'string' || rawText.length > 255) {
+                throw new BadRequestError({
+                    message: 'Invalid personalNoteText, possibly an i18n error.'
+                })
+            }
```

ESLint

[error] Missing semicolon.

(semi)